web/ui/react-app/src/App.css

```diff
@@ -10,6 +10,26 @@ button.classic-ui-btn {
     margin-bottom: 20px;
```

Comment:
GitHub doesn't let me comment on the right line, but you can now remove the body padding at the very top (line 1-3) to remove the extra space between the top of the page and the beginning of the navbar.